app/js/layout/App.jsx

```diff
@@ -59,6 +60,11 @@ class App extends Component {
     }
   }
 
+  shouldComponentUpdate(prevProps) {
+    return prevProps.isReady !== this.props.isReady ||
+      !_.isEqual(prevProps.profile, this.props.profile);
```

Comment:
Shouldn't you just have to move this to the `componentDidUpdate` `if`?

Reply:
Not sure to see what you mean